app/src/main/java/org/mozilla/fenix/home/HomeFragment.kt

```diff
@@ -161,6 +163,7 @@ class HomeFragment : Fragment() {
                 requireComponents.analytics.metrics.track(Event.OpenedAppFirstRun)
             }
         }
+        singleUseCollectionsTelemetryObserver()
```

Comment:
I'd prefer this to be in HomeActivity: the lifecycle of fragments is complicated and hard to predict so moving it into HomeActivity gives more me more confidence of correctness.

Reply:
done!